pkg/cmd/experimental/syncgroups/syncgroups.go

```diff
@@ -138,8 +139,8 @@ func NewCmdSyncGroups(name, fullName string, f *clientcmd.Factory, out io.Writer
 
 	cmd.Flags().StringVar(&whitelistFile, "whitelist", whitelistFile, "path to the group whitelist")
 	cmd.Flags().StringVar(&configFile, "sync-config", configFile, "path to the sync config")
-	cmd.Flags().StringVar(&typeArg, "type", typeArg, "type of group used to locate LDAP group UIDs: "+strings.Join(AllowedSourceTypes, ","))
-	cmd.Flags().BoolVar(&options.Confirm, "confirm", false, "if true, modify OpenShift groups; if false, display groups")
+	cmd.Flags().StringVar(&typeArg, "type", typeArg, "which groups whitelist entries refer to: "+strings.Join(AllowedSourceTypes, ","))
```

Comment:
`--names-from`?
`--which-groups `?

To be quite honest, I actually liked `--existing`, seeing as it succinctly indicates groups already existing in OpenShift.